lib/config.js

```diff
@@ -32,11 +32,23 @@ function expandTilde(apath) {
     return path.join(homedir, apath.slice(2));
```

Comment:
I want to use PM2 (https://github.com/Unitech/pm2) to launch Guerilla in a farm of machines.  WIth PM2 I need to either specify command line args, the  --config, or an env var as you suggest. 

My design goal is to have one env variable to rule them all. GuerillaHostName. Furthermore I want the configuration files (PM2's and Guerilla's) from git to be immutable-I don't want to modify them. Rather I want to download them and run. 

Possible solutions.

1.   I cannot use a single env variable, say GuerillaConfigFile for the config file per machine since some machines may startup both a master and a worker. I could use GuerillaConfigFileMaster and GuerillaConfigFileWorker env vars. They would apply depending on the mode (master/worker). This violates one flag to rule them all, but not by much:-)
1.  I could continue as is in this pull request, with the PM2 arguments specifying a config file with an embedded %var% which must be decoded by Guerilla
1. I could do the same via the PM2 env dict.  Again Guerilla would have to decode an embedded %var%.
1. I could massage the PM2 config files, essentially doing at pre-process time what I want to do in Guerilla at run time. This violates my design goal of having the config files be immutable
1. I could modify PM2. This might take some time and I don't know if they are receptive to it.